srcpkgs/bazel/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'bazel'
+pkgname=bazel
+version=3.5.0
+revision=1
+archs="x86_64"
+wrksrc="."
```

Comment:
You need to use `create_wrksrc` if the tarball doesn't contain a toplevel directory.

Reply:
What value do I put into it?

Reply:
Just `create_wrksrc=yes` is enough c: